### PR TITLE
multiply departure time

### PR DESCRIPTION
**Pull request by natesheehan**
RE schema update outlined in https://github.com/cyipt/actdev/pull/177

### CODE REVIEW

**Review by Robinlovelace — APPROVED** [review]

**Comment by Robinlovelace** [comment]
Looks good to me, minor change so merging directly. If this causes any other issues we'll pick up on them pretty soon. Thanks @natesheehan !

File: R/ab_scenario.R [hunk 1]
```diff
@@ -234,7 +234,7 @@ ab_save = function(x, f) {
 #' time_afternoon = ab_time_normal(hr = 17, sd = 0.75)
 #' as.POSIXct(trunc(Sys.time(), units="days") + time_afternoon)
 ab_time_normal = function(hr = 8.5, sd = 0.5, n = 1) {
-  round(stats::rnorm(n = n, mean = hr * 60^2, sd = sd * 60^2))
+  round(stats::rnorm(n = n, mean = hr * 60^2, sd = sd * 60^2)) * 10000.0
```
Comment (dabreegster) [inline R/ab_scenario.R:237]: This certainly gets the job done, but it breaks the documentation / API of `ab_time_normal`.

Why not make the change during serialization, at https://github.com/a-b-street/abstr/blob/0eb979d5c9d52ea1dce7f3c2aa01eeecfd389b4e/R/ab_scenario.R#L185? Semantically, we want to work in units of seconds everywhere, and just encode this special way in JSON.

Comment (Robinlovelace) [reply R/ab_scenario.R:237]: Good point Dustin, yes it makes sense to do this at the last possible moment, keeping them in more manageable units of seconds (not 10s of milliseconds?) while in memory.

Comment (Robinlovelace) [reply R/ab_scenario.R:237]: Plan to look at this now...

**Pull request merged**